config/application.rb

```diff
@@ -39,3 +39,14 @@ class Application < Rails::Application
     config.cas_url = "https://login.umd.edu/cas"
   end
 end
+
+module Blacklight::Solr
+  class Repository < Blacklight::AbstractRepository
+    protected
+      # set the SSL CA certs path for Solr connections
+      def build_connection
+        conn = Faraday.new ssl: { ca_path: Rails.configuration.ssl_ca_file }
```

Comment:
@peichman-umd should the SSL configuration be added conditionally if `Rails.configuration.ssl_ca_file` is present?  So that we do not need to configure `Rails.configuration.ssl_ca_file` when connecting to servers with a valid certificate. With this change, the application does not work correctly unless `Rails.configuration.ssl_ca_file` is configured - even when trying to connect to the solrdev server.

Reply:
@mohideen I agree. I will update this to only set up the custom connection if `ssl_ca_file` is present.

I am now actually doubting my implementation. It was working with fcrepo-vagrant, but I have since tried to run it again (with the same environment) and I am back to get the SSL certificate verification errors.